styles/lucid_lime/theme/collapsiblecategories.css

```diff
@@ -0,0 +1,46 @@
+/* Import base collapsible categories stylesheet */
+@import url('../../all/theme/collapsiblecategories.css');
+
+/* Styling overrides for Lucid Lime theme */
+
+/* Collapse button */
+.collapse-btn {
+	background-color: #262626;
+	border: 1px solid #b1e051;
+	margin: -19px 2px 0 0;
+}
+
+.collapse-btn:after,
+.collapse-btn:before {
+	background: #262626;
+	border: 1px solid #b1e051;
+}
+
+.collapse-btn:hover,
+.collapse-btn:hover:after,
+.collapse-btn:hover:before {
+	background-color: #353535; /* Button hover color */
+}
+
+/* Collapse show hide symbols */
+.collapse-hide,
+.collapse-show {
+	width: 14px;
+	height: 14px;
+}
+
+.collapse-hide:after,
+.collapse-show:before {
+	top: 5px;
+	left: 3px;
+	width: 4px;
+}
+
+.collapse-show:after {
+	left: 5px;
+	height: 4px;
+}
+
+.collapse-hide:before {
```

Comment:
This is unnecessary

Reply:
If I remove it then a dot appears in the top left corner 
![dot](https://user-images.githubusercontent.com/29129138/46244419-f677c300-c3de-11e8-8910-a5045234a57d.png)